### PR TITLE
Adding session to unsecure rest_api tests

### PR DESCRIPTION
**Pull request by Atri0**

### CODE REVIEW

**Comment by Atri0** [comment]
@denysord88 @agozhiy  Please review

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java [hunk 1]
```diff
@@ -57,59 +61,69 @@ public void checkLogsPage() {
             equalTo("sqlline.log")))
         .body("lastModified", hasItem(containsString(df.format(new Date()))));
   }
+
   @Test
-  public void DrillbitLogContent(){
+  public void DrillbitLogContent() {
```
Comment (denysord88) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Start method name from lower case please.

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java [hunk 2]
```diff
@@ -57,59 +61,69 @@ public void checkLogsPage() {
...
+        .get("/log/drillbit.log/content.json")
+        .then()
+        .statusCode(200)
+        .body("name", equalTo("drillbit.log"))
+        .body(containsString("lines"))
+        .body("maxLines", equalTo(10000));
   }
+
   @Test
-  public void DrillbitOutContent(){
```
Comment (denysord88) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Start method name from the lower case, please.

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java [hunk 3]
```diff
@@ -57,59 +61,69 @@ public void checkLogsPage() {
...
-      given()
-            .when()
-            .get("/log/drillbit.out/content.json")
-            .then()
-            .statusCode(200)
-            .body("name",equalTo("drillbit.out"))
-            .body(containsString("lines"))
-            .body("maxLines", equalTo(10000));
+  public void DrillbitOutContent() {
```
Comment (denysord88) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Start method name from the lower case, please.

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java [hunk 4]
```diff
@@ -57,59 +61,69 @@ public void checkLogsPage() {
...
-  public void DrillbitQueriesContent(){
-      given()
-            .when()
-            .get("/log/drillbit_queries.json/content.json")
-            .then()
-            .statusCode(200)
-            .body("name",equalTo("drillbit_queries.json"))
-            .body(containsString("lines"))
-            .body("maxLines", equalTo(10000));
+  public void DrillbitQueriesContent() {
```
Comment (denysord88) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Start method name from the lower case, please.

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java [hunk 5]
```diff
@@ -57,59 +61,69 @@ public void checkLogsPage() {
...
-  public void SqllineContent(){
-      given()
-            .when()
-            .get("/log/sqlline.log/content.json")
-            .then()
-            .statusCode(200)
-            .body("name",equalTo("sqlline.log"))
-            .body(containsString("lines"))
-            .body("maxLines", equalTo(10000));
+  public void SqllineContent() {
```
Comment (denysord88) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Start method name from the lower case, please.

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java [hunk 6]
```diff
@@ -57,59 +61,69 @@ public void checkLogsPage() {
...
-  public void SqllineQueriesContent(){
-      given()
-            .when()
-            .get("/log/sqlline_queries.json/content.json")
-            .then()
-            .statusCode(200)
-            .body("name",equalTo("sqlline_queries.json"))
-            .body(containsString("lines"))
-            .body("maxLines", equalTo(10000));
+  public void SqllineQueriesContent() {
```
Comment (denysord88) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Start method name from the lower case, please.

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/StorageTest.java [hunk 7]
```diff
@@ -46,7 +47,7 @@ public void testStoragePage() {
   @Test
   public void testStorageStoragePlugins() {
     //Get the response
-    Response response = get("/storage.json").then().extract().response();
+    Response response = given().filter(sessionFilter).when().get("/storage.json").then().extract().response();
```
Comment (denysord88) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/StorageTest.java]: Add new lines before ."method.." parts, please.

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java [hunk 8]
```diff
@@ -57,59 +61,69 @@ public void checkLogsPage() {
...
+  public void DrillbitLogContent() {
     given()
-            .when()
-            .get("/log/drillbit.log/content.json")
-            .then()
-            .statusCode(200)
-            .body("name",equalTo("drillbit.log"))
-            .body(containsString("lines"))
-            .body("maxLines", equalTo(10000));
+        .filter(sessionFilter)
```
Comment (denysord88) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: dfghdfgdfgdfg

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/StorageTest.java (hunk 7 above)
Comment (Atri0) [reply src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/StorageTest.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java (hunk 6 above)
Comment (Atri0) [reply src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java (hunk 5 above)
Comment (Atri0) [reply src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java (hunk 4 above)
Comment (Atri0) [reply src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java (hunk 3 above)
Comment (Atri0) [reply src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java (hunk 1 above)
Comment (Atri0) [reply src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java]: Done

**Comment by denysord88** [comment]
+1 LGTM

**Comment by agozhiy** [comment]
+1.

**Pull request merged**